src/styles/AnswerButton.scss

```diff
@@ -0,0 +1,23 @@
+.answer-button {
+    border: 1px solid white;
+    width: 100px;
+    height: 100px;
+}
+
+.answer-button-hovered {
+    border: 1px solid purple;
+}
+
+.answer-button-correct {
+    border: 1px solid black;
+    background-color: green;
+}
+
+.answer-button-wrong {
+    border: 1px solid black;
+    background-color: red;
+}
+
+.answer-button-selected {
+    border: 1px solid blue;
+}
```

Comment:
Jak będziesz stylował, to gotowe kolory są w `src/styles/_colors.scss`.
Różne gotowe style są w `/src/styles/_common.scss`, np. `border`, `border-radius` itp.

Reply:
Style do sprawdzenia dawałem, ostylować aplikację można później